primitives/staking/src/lib.rs

```diff
@@ -135,6 +135,32 @@ pub trait StakingInterface {
 		num_slashing_spans: u32,
 	) -> Result<bool, DispatchError>;
 
+	/// Get StakingLedger by controller account.
+	fn get_ledger(ctrl: &Self::AccountId) -> StakingLedger<Self>;
+
+	/// The ideal number of staking participants.
```

Comment:
The doc is very opaque, what is a staking participant? validator? nominator? active validator? validator candidate?

Reply:
This doc is copied from the staking pallet though, gotta fix both.